locale/en_US/manager.po

```diff
@@ -266,6 +266,21 @@ msgstr "Items can only be submitted by Editors and Section Editors."
 msgid "manager.sections.editors"
 msgstr "Section editors"
 
+msgid "manager.sections.inactive"
+msgstr "Inactive"
```

Comment:
Let's change this to `common.inactive` and move it to pkp-lib's `common.po` file. I suspect we'll get more use out of this in the future.